crowdkit/aggregation/classification/gold_majority_vote.py

```diff
@@ -13,7 +13,7 @@
 
 @attr.s
 class GoldMajorityVote(BaseClassificationAggregator):
-    r"""The **Gold Majority Vote** model is used when a golden dataset (ground truth) exists for some tasks.
+    r"""The **Gold Majority Vote** model is used when a golden [dataset](https://toloka.ai/datasets) (ground truth) exists for some tasks.
```

Comment:
This link does not contribute to the documentation, please remove it. Below you have really good examples of links, but this one is too much.